repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -355,10 +355,10 @@ private void mainProcess() {
 
         ProjectInspector inspector;
 
-        if (config.getLauncherMode() == LauncherMode.BEARS) {
-            inspector = new ProjectInspector4Bears(buildToBeInspected, this.config.getWorkspacePath(), serializers, this.notifiers);
-        } else {
+        if (config.getLauncherMode() == LauncherMode.REPAIR) {
```

Comment:
Why did you change that if? It seems to me more interesting to check if it's bears or not: the default case should be repair...

Reply:
Because in line 344 there is an if for `LauncherMode.REPAIR`. I just wanted to make the two ifs to match to be easier to understand the code.

Reply:
I changed the if back, and I put the other if comparing with `LauncherMode.BEARS` then.